[CIVIS-3137] Replace wall time documentation with new metric "total sessions duration" (#20486)
DataDog/documentation: content/en/continuous_integration/troubleshooting.md

4. If no CI provider environment variables are found, tests results are sent with no Git metadata.

> ### The tests total time is empty

Reviewer: ```suggestion
### The total test time is empty
```

> If you cannot see the tests total time it is likely that test suite level visibility is not enabled. To confirm this is the case, check if your language supports test suite level visibility in [Supported features][15]. If it is supported, update your tracer to the latest version.

Reviewer: ```suggestion
If you cannot see the total test time, it is likely that test suite level visibility is not enabled. To confirm, check if your language supports test suite level visibility in [Supported features][15]. If test suite level visibility is supported, update your tracer to the latest version.
```

If after updating the tracer version you still don't see the total time, contact [Datadog support][2] for help.

> ### The tests tests total time is not what is expected

Reviewer: ```suggestion
### The total test time is different than expected
```

#### How total time is calculated

> The total time is defined as the total sum of the maximum test session durations.

Reviewer: ```suggestion
The total time is defined as the sum of the maximum test session durations.
```
